### PR TITLE
[dev] Add Provides net-snmp-utils

### PR DESCRIPTION
**Pull request by henryli001**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Add minor provides for net-snmp-utils

###### Change Log  

- Provides net-snmp-utils from net-snmp

###### Does this affect the toolchain?  

**YES**
NO

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/...

###### Test Methodology

- Local Build

### CODE REVIEW

File: SPECS/net-snmp/net-snmp.spec [hunk 1]
```diff
@@ -17,6 +17,7 @@ BuildRequires:  perl
 BuildRequires:  systemd
 Requires:       perl(:MODULE_COMPAT_%(eval "`perl -V:version`"; echo $version))
 Requires:       systemd
+Provides:       net-snmp-utils = %{version}-%{release}
```
Comment (schmittjoseph) [inline SPECS/net-snmp/net-snmp.spec:20]: Replace `net-snmp` with `%{name}`

File: SPECS/net-snmp/net-snmp.spec [hunk 2]
```diff
@@ -96,6 +97,8 @@ rm -rf %{buildroot}/*
 %exclude /usr/lib/perl5/perllocal.pod
 
 %changelog
+*   Thu Feb 04 2021 Henry Li <lihl@microsoft.com> 5.8-6
```
Comment (schmittjoseph) [inline SPECS/net-snmp/net-snmp.spec:100]: Please also run the `spec-cleaner` on this SPEC, the changelog should be in a slightly different format.

**Comment by schmittjoseph** [comment]
minor: for dev prs, please start the PR with `[dev]` so they are easily distinguished.

**Comment by schmittjoseph** [comment]
minor: please also fill out the PR template / remove fields that don't apply. See https://github.com/microsoft/CBL-Mariner/pull/581 for an example

**Review by schmittjoseph — CHANGES_REQUESTED** [review]

File: SPECS/net-snmp/net-snmp.spec (hunk 1 above)
Comment (henryli001) [reply SPECS/net-snmp/net-snmp.spec]: Fixed

File: SPECS/net-snmp/net-snmp.spec (hunk 2 above)
Comment (henryli001) [reply SPECS/net-snmp/net-snmp.spec]: Fixed

File: SPECS/net-snmp/net-snmp.spec [hunk 3]
```diff
@@ -1,29 +1,31 @@
 %global __requires_exclude perl\\(.*\\)
+%define sha1 net-snmp=78f70731df9dcdb13fe8f60eb7d80d7583da4d2c
```
Comment (schmittjoseph) [inline SPECS/net-snmp/net-snmp.spec:2]: nit: please also remove the `sha1` while you're in this spec

**Review by schmittjoseph — APPROVED** [review]

**Pull request merged**